scripts/verify-hashes

```diff
@@ -3,96 +3,69 @@
 import os
 import sys
 
-if len(sys.argv) != 2:
-    print("Usage: ./scripts/verify-hashes path/to/sha256sums.txt")
-    sys.exit(1)
-
-requirements_file = "requirements.txt"
-if not os.path.exists(requirements_file):
-    requirements_file = "requirements/requirements.txt"
-    if not os.path.exists(requirements_file):
-        print("Cannot find requirements.txt")
-        sys.exit(1)
-
-# This is the already gpg signed and verified data
-sha256sum_data = {}
-with open(sys.argv[1]) as fobj:
-    data = fobj.readlines()
-
-for line in data:
-    line = line.strip()
-    if line.startswith('#'):
-        continue
-    words = line.split()
-    # just check that the sums are of correct length
-    if len(words[0]) != 64:
-        print("Wrong sha256sum {0}".format(words[0]))
-        sys.exit(3)
-    sha256sum_data[words[0]] = True
-
-
-# Now read the requirements.txt file
-lines = []
-with open(requirements_file) as fobj:
-    lines = fobj.readlines()
-
-# Now we want to verify that for each dependency in the project
-# to be packaged, has a matching source tarball on FPF's PyPI.
-
-# Remove lines with comments.
-uncommented_lines = [line for line in lines if not line.lstrip().startswith('#')]
-
-# The hashes for a given requirement will be distributed
-# across multiple lines, e.g.
-#
-# atomicwrites==1.2.1 \
-#    --hash=sha256:0312ad34fcad8fac3704d441f7b317e50af620823353ec657a53e981f92920c0 \
-#    --hash=sha256:ec9ae8adaae229e4f8446952d204a3e4b5fdd2d099f9be3aaf556120135fb3ee
-#
-# Let's create a list with one list element per dependency.
-
-dependencies_with_hashes = ''.join(uncommented_lines).replace('\\\n', '').splitlines()
-
-# Now we'll construct a dict containing each dependency,
-# and a list of its hashes, e.g.:
-#
-# {
-#   'alembic': ['04bcb970ca8659c3607ddd8ffd86cc9d6a99661c9bc590955e8813c66bfa582b']
-# }
+from pathlib import Path
+import utils
+
+try:
+    import tomllib
+except ModuleNotFoundError:
+    # pre-Python 3.11 compatibility
+    import toml as tomllib
```

Comment:
Same issue here. Maybe it would be easier if we did `from utils import tomllib` instead of reimplementing the try/catch.

Reply:
Turns out we're not even using the import here, it all got moved into utils :sweat_smile:. Removed.